.github/workflows/continuous-integration.yml

```diff
@@ -39,18 +39,20 @@ jobs:
           path: ~/conda_pkgs_dir
           key: ${{ runner.os }}-conda-${{ hashFiles('**/environment.yml') }}
           restore-keys: ${{ runner.os }}-conda-
-      - name: Set up pip cache
-        uses: actions/cache@v4
+      - name: Set up uv
+        uses: astral-sh/setup-uv@v4
         with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('pyproject.toml') }}
-          restore-keys: ${{ runner.os }}-pip-
+          version: "0.5.*" 
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
```

Comment:
I _think_ these are enabled by default, so can be removed?

```suggestion
```

Reply:
I think you are right, though it is not obvious in the docs!

In the setup-uv `action.yml` the default setting for `enable-cache` is `auto`: https://github.com/astral-sh/setup-uv/blob/main/action.yml#L21-L23

It will be switched to `true` if the action is running on a github hosted runner: https://github.com/astral-sh/setup-uv/blob/85aa0bf0c11c139baabd61df94702a838ff76d62/src/utils/inputs.ts#L21

But the default for `cache-dependency-glob` is indeed good enough: https://github.com/astral-sh/setup-uv/blob/main/action.yml#L24-L30